torch/ao/quantization/fx/_lower_to_native_backend.py

```diff
@@ -28,7 +28,7 @@
 )
 
 
-QOP_TO_ARG_NAMES_TO_SKIP = {
+QOP_TO_ARG_NAMES_TO_SKIP: dict[Callable, list[str]] = {
```

Comment:
Would it not be better to do `dict[OpOverloadPacket, list[str]]`?

Reply:
That would be great, but the place where we actually query the dictionary can only assert that the key is `Callable[..., Any]` at best, so this typing needs to be more generic to avoid triggering warnings about that.